packages/front-end/pages/experiments/index.tsx

```diff
@@ -447,236 +450,274 @@ const ExperimentsPage = (): React.ReactElement => {
-
-                          return (
-                            <button
-                              key={tab}
-                              className={clsx("border mb-0", {
-                                "badge-purple font-weight-bold": active,
-                                "bg-white text-secondary": !active,
-                                "rounded-left": i === 0,
-                                "rounded-right":
-                                  tab === "archived" ||
-                                  (tab === "stopped" && !hasArchived),
-                              })}
-                              style={{
-                                fontSize: "1em",
-                                opacity: active ? 1 : 0.8,
-                                padding: "6px 12px",
-                              }}
-                              onClick={(e) => {
-                                e.preventDefault();
-                                onToggleTab(tab)();
-                              }}
-                              title={
-                                active && tabs.length > 1
-                                  ? `Hide ${tab} experiments`
-                                  : active
-                                  ? `Remove filter`
-                                  : tabs.length === 0
-                                  ? `View only ${tab} experiments`
-                                  : `Include ${tab} experiments`
-                              }
-                            >
-                              <span className="mr-1 ml-2">
-                                {tab.slice(0, 1).toUpperCase()}
-                                {tab.slice(1)}
-                              </span>
-                              {tab !== "archived" && (
-                                <span className="badge bg-white border text-dark mr-2 mb-0">
-                                  {tabCounts[tab] || 0}
-                                </span>
-                              )}
-                            </button>
-                          );
-                        }
-                      )}
-                    </div>
-                    <div className="col-auto">
-                      <Field
-                        placeholder="Search..."
-                        type="search"
-                        {...searchInputProps}
-                      />
-                    </div>
-                    <div className="col-auto">
-                      <TagsFilter filter={tagsFilter} items={items} />
-                    </div>
-                    <div className="col-auto">
-                      <Link
-                        href="https://docs.growthbook.io/using/growthbook-best-practices#syntax-search"
-                        target="_blank"
-                      >
-                        <Tooltip body={searchTermFilterExplainations}></Tooltip>
+              )) ||
```

Comment:
Nit: The `||` syntax is not usually used like this when rendering components. Ternary expressions are more common.